packages/@o3r/eslint-config/src/rules/typescript/eslint.cjs

```diff
@@ -105,12 +105,6 @@ const config = [
       ],
       'no-unreachable': 'error',
       'no-unsafe-negation': 'error',
-      'no-use-before-define': [
```

Comment:
does it means that it is `error` without exception or `off`?

Reply:
No it's disabled by default